linkerd/app/core/src/errors.rs

```diff
@@ -9,12 +9,15 @@ use linkerd_proxy_http::{ClientHandle, HasH2Reason};
 use linkerd_timeout::{FailFastError, ResponseTimeout};
 use linkerd_tls as tls;
 use pin_project::pin_project;
+use std::fmt;
 use std::pin::Pin;
 use std::task::{Context, Poll};
 use thiserror::Error;
 use tonic::{self as grpc, Code};
 use tracing::{debug, warn};
 
+pub const L5D_HTTP_ERROR_MESSAGE: &str = "l5d-http-error-message";
```

Comment:
Let's call this `l5d-proxy-error` -- I don't think _http_ is particularly meaningful (this is set on HTTP requests) and _message_ is superfluous. This header indicates that there was a proxy error.